### DISCUSSION TITLE
Rename value in all .json file (metadata)

### DISCUSSION THREAD

**Discussion by DreamEp** [nftchef/art-engine#112] [Help + Q&A]
Hey everyone ! 

Is there a way to rename a value from a trait type in all my metadata .json file.
For exemple : 
{
      "trait_type": "Clothes",
      "value": "SuitForType2"
}
to : 
{
      "trait_type": "Clothes",
      "value": "Suit"
}

Thank you in advance ! :)

**Comment by nftchef** [comment]
The easiest way is to use the built-in find and replace all feature in VSCode (or other text editor)
